### PR TITLE
Build - fixed Windows compatibility and added build script

### PR DESCRIPTION
**Pull request by stephendade**
Fixed up to compile/install/run in Windows sucessfully.

This comes out of some discussions in Ardupilot - The Ardupilot team has recently added support for the Morse simulator (https://discuss.ardupilot.org/t/morse-3d-simulator-with-ardupilot/35930)

### CODE REVIEW

**Comment by severin-lemaignan** [comment]
Hi and thanks for the PR @stephendade!

Very happy to merge this patch, but could you first update the doc? at least [installation.rst](https://github.com/morse-simulator/morse/blob/master/doc/morse/user/installation.rst) and [INSTALL](https://github.com/morse-simulator/morse/blob/master/INSTALL) -- and if additional notes are required for Windows users beyond installation, please do not hesitate to add them.

Please also add a line into the [RELEASE_NOTES](https://github.com/morse-simulator/morse/blob/master/RELEASE_NOTES).

**Comment by stephendade** [comment]
Cool - I've added in the documentation updates

File: INSTALL [hunk 1]
```diff
@@ -168,6 +173,26 @@ When updating MORSE to a more recent version, you'll simply have to do::
...
 
+Windows
+-------
+
+Download the latest version of the source code. It is stored in a git
+repository::
+
+  $ git clone https://github.com/morse-simulator/morse.git
+
+The MORSE_BLENDER environment variable should be set the the location and filename
```
Comment (dgerod) [inline INSTALL]: There is written two times "the" between "be set" and "location and filename".

Comment (stephendade) [reply INSTALL]: Fixed!

**Comment by severin-lemaignan** [comment]
Great, thanks!

**Pull request merged**